1.0/profile.ttl

```diff
@@ -0,0 +1,63 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX prof: <http://www.w3.org/ns/dx/prof/>
+PREFIX role: <http://www.w3.org/ns/dx/prof/role/>
+PREFIX sdo: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+<http://www.opengis.net/def/geosparql>
+    a prof:Profile ;
+    dcterms:identifier "geosparql" ;
+    dcterms:created "2009-10-27"^^xsd:date ;
+    dcterms:modified "2012-07-19"^^xsd:date ;
+    dcterms:issued "2020-xx-xx"^^xsd:date ;
+    dcterms:publisher <https://www.ogc.org> ;
+    dcterms:title "GeoSPARQL Specification"@en ;
```

Comment:
Again, (if I understand correctly) the resource that is described is a profile of GeoSPARQL, not GeoSPARQL itself. So I would expect a title like "Profile of GeoSPARQL 1.0".

Reply:
Updated as you suggest. Sorry, it does look like I did reuse a lot of properties from GeoSPARQL ont without sufficint modification...